service/k8sclient/k8sclient.go

```diff
@@ -1622,13 +1623,27 @@ func (c *K8sClient) GetConsumedDiskBytes(ctx context.Context, clusterType kubern
 		if err != nil {
 			return 0, errors.Wrap(err, "can't compute consumed disk size: failed to get worker nodes")
 		}
+		clientConfig, err := clientcmd.NewClientConfigFromBytes([]byte(c.kubeconfig))
+		if err != nil {
+			return 0, errors.Wrap(err, "failed to build kubeconfig out of given path")
+		}
+		config, err := clientConfig.ClientConfig()
+		if err != nil {
+			return 0, errors.Wrap(err, "failed to build kubeconfig out of given path")
+		}
+		clientset, err := kubernetes.NewForConfig(config)
+		if err != nil {
+			return 0, errors.Wrap(err, "failed to build client out of submited kubeconfig")
+		}
```

Comment:
What if we move it to `New`?

Reply:
I would do it if we move to client-go based approach entirely. Right now it'd be unnecessary overhead when we use it only in one place.